### PR TITLE
Fix documentation warnings from Sphinx 1.6.7

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Fixes [Sphinx warnings](https://builds.mantidproject.org/job/pull_requests-ubuntu/32079/sphinx/) from version 1.6.7 after moving pull request builds to Ubuntu 18.04.

**To test:**

Code review is sufficient.

*There is no associated issue.*

*This does not require release notes* because **it is an internal issue.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by martyngigg** [comment]
@mantidproject/gatekeepers Any chance of just merging this without a second review? The Ubuntu pull requests will keep failing with these warnings until we get it in.

**Review by AndreiSavici — APPROVED** [review]

**Pull request merged**
